### ISSUE TITLE
Playtime clock not working on NES suspend points

### ISSUE THREAD

**Issue by hovdogg** [KMFDManic/NESC-SNESC-Modifications#138]
I'm using Hakchi2 CE v1.2.5 and km_RetroArch 173 Xtreme NetPLAY Edition on an NES Classic.  I only have NES games on it.  For some reason, the suspend points I make in any game almost always register as 0:00 on the playtime clock no matter how long I've been playing--it doesn't matter if I am using Nestopia or Fceumm.  Occasionally, an old suspend point will add a minute or two if I load and play from it a few times.  I've tried uninstalling and resetting to factory settings and then reinstalling a number of times to no avail.  I also have an SNES classic, which is using the same version of retroarch, but that doesn't have this issue at all, and as far as I know, I haven't done anything differently to that one (other than only running SNES games on it.)

When I use Cluster's version of Retroarch, suspend points in Retroarch work fine.  I've looked and haven't found anyone talking about this issue, so hopefully it hasn't already been answered before.  Thanks.

**Comment by KMFDManic** [comment]
Test with /bin/fceumm or /bin/nestopia instead of /bin/nes

You can also try /bin/quicknes

**Comment by hovdogg** [comment]
I tested each one and they all have the same issue.  Still 0:00 on the playtime clock.

**Comment by KMFDManic** [comment]
Are you, by chance, running the NEW NES Classic?

**Comment by hovdogg** [comment]
Yes, I am.

**Comment by hovdogg** [comment]
I am using a new NES classic.  Oddly, trying to run a game with /bin/nes still launches retroarch and plays the game with fceumm.  I tried doing a fresh install of the custom kernel and retroarch, and it still did this.  /bin/nestopia and /bin/quicknes both launch the correct core, but bin nes seems to be defaulting to fceumm.

Here's the command line if that helps at all: /bin/nes /usr/share/games/nes/kachikachi/CLV-H-YIWQJ/Battletoads.nes.7z

**Comment by KMFDManic** [comment]
@hovdogg I didn't forget about you.  The biggest thing that has changed between Cluster's RetroArch and Xtreme and NEO is the fact that "saves are more optimally reliable and work for GBA/N64, where they do not on the previous RetroArch.  These changes might have caused the side effect of the lost time played on clock.  You can use suspend states on N64 Games, which you cannot do on old RetroArch!  And, many GBA Games did not save at all!  This was a very important change!  I can poke into it a little more.  But, it might be a tricky scenario due to how drastic the entire change was, save state wise.  But, that is one of the key differences!  Saves:)

**Comment by KMFDManic** [comment]
If you still need help, let me know.  But, it seems the playtime clock may possibly only function with Kachikachi, not with RetroArch Cores.  Have you tried going into RetroArch Settings to see if you are in a Core, or simply Kachikachi?

**Comment by hovdogg** [comment]
It's all good.  Sorry for taking awhile to respond; my life has been a little crazy the last couple weeks.  I was only interested in playing NES games, so I just went back to Cluster's retroarch.  I was wanting to use the newer retroarch, so I could get the NES-crt filter on there, but I played around with the settings in Cluster's retroarch to get it to a crt look that I like, so I'll stick with that.  I get that being able to use suspend points on n64 and some of the other problems fixed are definitely more important than a little time clock.  Thanks for your help, though!
